.github/workflows/tests.yml

```diff
@@ -11,18 +11,18 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                php: [ "8.0", "8.1", "8.2", "8.3" ]
+                php: [ "8.1", "8.2", "8.3" ]
                 extensions: [""]
                 tools: [""]
                 composerflags: [""]
                 include:
-                  - php: "8.0"
+                  - php: "8.1"
                     extensions: "protobuf,grpc"
                     tools: "pecl"
                   - php: "8.3"
```

Comment:
Let's increment this to PHP 8.4 (and we will need to update `sync-repo-settings.yaml` as well)

Reply:
added in [44be7ac](https://github.com/googleapis/gax-php/pull/598/commits/44be7ac778e6e5fb4f776bc18b1f386b7affbf2c)